data/available_datasets.json

```diff
@@ -1,6 +1,162 @@
+                    "type": "code",
+                    "repo_name": "Zenodo",
+                    "accession_number": "",
+                    "description": "Analyses were conducted using STATA 16, the code to replicate the analyses in the article",
+                    "data_type": [
+                        "Social science and humanities data",
+                        "Other data"
+                    ],
+                    "data_url": "https://zenodo.org/record/7214856#.Y9obkezMKX0"
+                }
+            ],
+            "title": "Financial incentives for vaccination do not have negative unintended consequences",
+            "issue": "7944",
+            "volume": "613",
+            "publisher": "Springer Science and Business Media LLC",
+            "published": "2023-01-11",
+            "author": [
+                "Schneider F. H.",
+                "Campos-Mercade P.",
+                "Meier S.",
+                "Pope D.",
+                "Wengström E."
+            ]
+        },
+        {
+            "doi": "10.1371/journal.pone.0280355",
+            "available_items": [
+                {
+                    "type": "data",
+                    "repo_name": "Github",
+                    "accession_number": "",
+                    "description": "Aggregated data of COVID-19 mortality and long-term air pollution concentrations",
+                    "data_type": [
+                        "Public health data",
+                        "Other data"
+                    ],
+                    "data_url": "https://github.com/LopezFeldman/Airpollution_Covid19"
+                },
+                {
+                    "type": "code",
+                    "repo_name": "Github",
+                    "accession_number": "",
+                    "description": "Stata scripts used to compile the statitics",
+                    "data_type": [
+                        "Public health data",
+                        "Other data"
+                    ],
+                    "data_url": "https://github.com/LopezFeldman/Airpollution_Covid19"
+                }
+            ],
+            "title": "Association between long-term air pollution exposure and COVID-19 mortality in Latin America",
+            "issue": "1",
+            "volume": "18",
+            "publisher": "Public Library of Science (PLoS)",
+            "published": "2023-01-17",
+            "author": [
+                "Bonilla J. A.",
+                "Lopez-Feldman A.",
+                "Pereda P. C.",
+                "Rivera N. M.",
+                "Ruiz-Tagle J. C."
+            ]
+        },
+        {
+            "doi": "10.3389/fimmu.2022.1079995",
+            "available_items": [
+                {
+                    "type": "data",
+                    "repo_name": "NCBI",
+                    "accession_number": "PRJNA900274",
+                    "description": "All genomic dataset from this project are available under a NCBI BioProject",
```

Comment:
All genomic datasets from this project are available as part of an NCBI BioProject